From 23d9f26a7c0a03f4625dc8d4d04405a3cb779842 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Tue, 4 Jun 2019 21:27:33 +0200
Subject: ThrottlingController::Liveness needs to be uint32_t

We are setting kAlive and kDead values assigning values that
are bigger than the maximum signed int32. It is better to use
uint32_t in this case.

Bug: 819294
Change-Id: If72b48291a66a3a9db24b4c8e2d11d31936a66ee

diff --git a/services/network/throttling/throttling_controller.h b/services/network/throttling/throttling_controller.h
index 43751c4976a4..3c6f87be850c 100644
--- a/services/network/throttling/throttling_controller.h
+++ b/services/network/throttling/throttling_controller.h
@@ -38,7 +38,7 @@ class COMPONENT_EXPORT(NETWORK_SERVICE) ThrottlingController {
 
   // TODO(https://crbug.com/960874): Debugging code to try and shed some light
   // on why the owned maps are invalid.
-  enum class Liveness : int32_t {
+  enum class Liveness : uint32_t {
     kAlive = 0xCA11AB13,
     kDead = 0xDEADBEEF,
   };
-- 
2.20.1

